Tasks/MavenV3/task.json

```diff
@@ -84,7 +84,7 @@
             "name": "testResultsFiles",
             "type": "filePath",
             "label": "Test results files",
-            "defaultValue": "**/TEST-*.xml",
+            "defaultValue": "**/surefire-reports/Test-*.xml",
```

Comment:
I think "Test" should be uppercase "TEST".  At least I thought that was the surefire default capitalization.